pkg/pss/pss.go

```diff
@@ -129,14 +128,17 @@ func (p *pss) topics() []Topic {
 }
 
 // TryUnwrap allows unwrapping a chunk as a trojan message and calling its handlers based on the topic.
-func (p *pss) TryUnwrap(ctx context.Context, c swarm.Chunk) error {
+func (p *pss) TryUnwrap(ctx context.Context, c swarm.Chunk) {
+	if len(c.Data()) < 4104 {
```

Comment:
Could we create constant for `4104` ?

Reply:
it would be better to extract this to a `var` which is calculated once, since it is not trivial to assume the reader knows why this specific number was selected (such as in the previous check: `trojanChunkMinDataLen = swarm.SpanSize + NonceSize + TopicSize + LengthSize`)